samples/bluetooth/conn_time_sync/README.rst

```diff
@@ -0,0 +1,210 @@
+Therefore the peripheral(s) can obtain use its local anchor point information from later connection events for clock synchronization.
+
+The figure below shows how the peripheral obtains the local peripheral trigger time for a given central clock trigger time.
+The peripheral uses provided central clock synchronization information together with a local anchor point information.
+
+.. figure:: images/TimeSync.svg
+   :alt: Clock synchronization
+
+   Clock synchronization between central and peripheral.
+
+
+Timed trigger accuracy
+======================
+
+There are multiple factors that influence the accuracy of timed trigger.
+
+#. The time since the last clock synchronization.
+#. How much the central and peripheral clock drift apart.
+#. How much the central and peripheral clock jitter.
+#. How accurately the timed activity is triggered.
+
+In this sample the peripheral clocks are synchronized every time new anchor point information is available.
+That is, every connection event where the peripheral successfully receives a packet from the central.
+The expected time between clock synchronizations is therefore equal to the configured connection interval of 10 milliseconds.
+
+The Bluetooth specification defines a worst case allowed combined sleep clock accuracy of 1000 ppm.
+That is, for each second the clocks may drift at maximum 1 ms.
+The development kits are equipped with a low frequency clock crystal with an accuracy in the range of 20 to 100 ppm.
+
+The Bluetooth specification defines a worst case allowed sleep clock jitter of 16 us.
+The SoftDevice Controller anchor points jitters less, see :c:macro:`MPSL_TIMESLOT_START_JITTER_US`.
+
+The timed trigger (toggling of a LED) is in this sample application triggered using a microsecond accurate timer.
+This gives an accuracy of 0.5 us.
+
+The worst case offset between the central trigger and peripheral trigger occurs when all the components mentioned above drift and jitter in different directions.
+In many cases the offset will be less.
+
+.. code-block:: none
+
+  worst_case_offset = drift + clock_jitter + trigger_jitter
+  worst_case_offset = (c_sca_ppm + p_sca_ppm) * conn_interval + c_clock_jitter + p_clock_jitter + c_trigger_accuracy + p_trigger_accuracy
+  worst_case_offset = (50 ppm + 50 ppm) * 10 ms + 1 us + 1 us + 0.5 us + 0.5 us
+  worst_case_offset = 1 us + 2 us + 1 us
+  worst_case_offset = 4 us
+
+.. note::
+   This sample requires less hardware resources when it is run on an nRF54L Series device compared to the nRF52 or nRF53 Series devices.
+   On an nRF54L Series device, only one GRTC channel and PPI channel is needed to set up accurate toggling of an LED.
+   On nRF52 and nRF53 Series devices, you also need one RTC peripheral, one TIMER peripheral, one EGU channel, three PPI channels, and one PPI group.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/iso_time_sync`
```

Comment:
What is the intention with this? It shows up as nothing: https://github.com/rugeGerritsen/sdk-nrf/blob/conn-time-sync/samples/bluetooth/conn_time_sync/README.rst#building-and-running